makefile

```diff
@@ -42,14 +42,6 @@ help:
 	@echo '  To run a single header test, add "-test" to the end of the file name.'
 	@echo '  Example: make stan/math/constants.hpp-test'
 	@echo ''
-	@echo '  - test-math-dependencies : walks through all the header files and indicates'
-	@echo '      when the math dependencies are violated. Dependencies should follow:'
-	@echo '      * rev -> prim'
-	@echo '      * fwd -> prim'
-	@echo '      * mix -> {rev, fwd, prim}'
-	@echo '      * within {prim, rev, fwd, mix}: mat -> arr -> scal'
-	@echo '      * only include {prim, rev, fwd, mix}/meta.hpp from the meta subfolders'
-	@echo ''
```

Comment:
Instead of removing `make test-math-dependencies` can we have that call run the python script?

Reply:
That is a great idea, I dont have to change what Jenkins runs then also. Will do.